index.html

```diff
@@ -50,7 +50,7 @@ <h1>I’m Tuyishimire Eric Glad to see you!</h1>
                 <div class="grid-item-photo" id="photo1">
                 </div>
                 <div class="grid-item-section">
-                    <h1 class="title">Tonic</h1>
```

Comment:
After reviewing the code, we identified inconsistencies in naming headings. We changed heading naming in the works section from "h1" headings to "h2" headings. This is to ensure that the texts are easily accessible and readable as well.